ChangeLog.md

```diff
@@ -20,6 +20,10 @@ See docs/process.md for more on how version tagging works.
 
 3.1.52 (in development)
 -----------------------
+- The `ERRNO_SUPPORT` setting is now deprecated since the `setErrNo` function
+  that it controls is no longer used by emscripten.  Use of `errno` from in C
+  code is largely unaffected by this change since it (mostly) did not honor this
+  setting. (#21074)
```

Comment:
This text wasn't clear to me on first read. How about this:

"The `SUPPORT_ERRNO` setting is now deprecated, as it only controlled setting `errno` from JS library functions, and after moving code from there to wasm no JS library code refers to `errno` any more."